NEWS.md

```diff
@@ -44,6 +44,8 @@ based on [Common Terminology Criteria for Adverse Events (CTCAE) v5.0](https://c
 
 ## Updates of Existing Functions
 
+- Function `derive_param_first_event` has been replaced by a more generalized `derive_param_extreme_event` function with new argument `mode` allowing for the selection of either the `"first"` or `"last"` event record according to the conditions provided. Also the `date_var` argument has been replaced with the `order` argument instead. In addition, three new arguments `new_var`, `true_value`, and `false_value` have been added to allow the user to choose what variable is used to indicate whether an event happened, and the values it is given (#1317) (#1242).
```

Comment:
Please add brackets to function names, i.e., `derive_parm_first_event()` instead of `derive_param_first_event`. Then links are created in the html documentation.